crates/web-plugins/didcomm-messaging/protocols/pickup/src/handler.rs

```diff
@@ -489,7 +557,7 @@ mod tests {
             .await
             .unwrap_err();
 
-        assert_eq!(error, PickupError::MissingClientConnection);
+        assert_eq!(error.to_string(), "Failed to retrieve client connection");
```

Comment:
Just add a new assertion without removing the pre-existing one that was actually testing for the type of the error.